src/test/java/racinggame/car/CarsTest.java

```diff
@@ -0,0 +1,33 @@
+package racinggame.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racinggame.ui.ResultView;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class CarsTest {
+
+    @DisplayName("자동차 레이싱 우승자 출력하기")
+    @Test
+    void winner() {
+        Car car1 = new Car(CarName.of("car1"));
+        Car car2 = new Car(CarName.of("car2"));
+        Car car3 = new Car(CarName.of("car3"));
+        car1.race(() -> true);
```

Comment:
테스트 잘해주셨습니다 😄 
테스트 Stub 이라는게 있습니다 
이렇게 익명함수로 하셔도 되는데 
자주 테스트에 사용될 수 있도록 늘 예상하는 값을 주는 Stub 객체를 만들어서 테스트 하는 방법인데요
핵심은 **"변화의 요인은 통제하고, 내가 구현한 부분을 테스트 하기 위함"** 입니다 
 
여기에서는 사실 이렇게 하셔도 되긴 합니다만 
이 참에 개념도 한번 정리하고 가시면 좋을 것 같아서 의견 드렸습니다 😄 
```java
public class AlwaysGo implements MovingStrategy {
    @Override
    public boolean isMove() {
        return true;
    }
}
``` 

```suggestion
        car1.race(new AlwaysGo());
```
도움 되실 만한 내용 공유 드립니다 🙇 
> https://medium.com/@SlackBeck/%ED%85%8C%EC%8A%A4%ED%8A%B8-%EC%8A%A4%ED%85%81-test-stub-%EC%9D%B4%EB%9E%80-%EB%AC%B4%EC%97%87%EC%9D%B8%EA%B0%80-ff9c8840c1b0

Reply:
오... 좋은 자료 공유 감사합니다!